docs/user-manuals/imagepulljob.md

```diff
@@ -97,6 +105,39 @@ spec:
       io.kubernetes.image.app: "foo"
 ```
 
+## ImageListPullJob
+
+**FEATURE STATE:** Kruise v1.5.0
+
+ImagePullJob can only support a single image pre-download, in order to meet the pre-download requirements of more images,
+ImageListPullJob can support multiple images pre-download, its fields are basically the same as ImagePullJob, except images. as follows:
+
```

Comment:
ImagePullJob can only support a single image pre-download, one can use multiple ImagePullJob to download multiple images, or use ImageListPullJob to pre-download multiple images in a single job.